src/index.ts

```diff
@@ -1,3 +1,4 @@
+import type { EmulatorOptions } from 'andromatic';
```

Comment:
`Ctrl + C`ing the analysis program sometimes works, but sometimes it doesn't do anything.

I'd guess we need to update our `SIGINT` handler?

And while you're at it, we should also only remove our handler instead of doing `removeAllListeners()`, which would remove the need for the `noSigint` option.